### DISCUSSION TITLE
How events work with private/public transaction in Go Quorum?

### DISCUSSION THREAD

**Discussion by ghost** [ConsenSys/quorum#1560] [Q&A] [answered]
Hi, I want to use events with private transactions So please provide me any link or any example?

**Comment by baptiste-b-pegasys** [comment, marked as answer]
Hi, here is a link for events and logs:
https://media.consensys.net/technical-introduction-to-events-and-logs-in-ethereum-a074d65dd61e

**Reply by antonydenyer** [reply]
Under the hood GoQuorum will take care of it for you. If you are a party to the private transaction you will see the event.
